### PR TITLE
DOC-10375 SDK link updates & tidy

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/fts/pages/fts-searching-with-sdk.adoc [hunk 1]
```diff
@@ -13,54 +13,57 @@ Couchbase provides several SDKs to allow applications to access a Couchbase clus
...
+
+|Go SDK
+|The Couchbase Go SDK allows you to connect to a Couchbase Server cluster from Go.
+The Go SDK is a native Go library and uses the high-performance gocbcore to handle communicating to the cluster over Couchbase's binary protocols.
+|xref:2.5@go-sdk:howtosfull-text-search.adoc[Go SDK 2.5]
+
+| Java SDK
+| The Java SDK forms the cornerstone of our JVM clients.
+It allows Java applications to access a Couchbase Server cluster.
+The Java SDK offers traditional synchronous APIs and scalable asynchronous APIs to maximize performance
```
Comment (simon-dew) [inline modules/fts/pages/fts-searching-with-sdk.adoc]: Full stop after "performance"? All the other Detail entries have a full stop.

File: modules/fts/pages/fts-searching-with-sdk.adoc [hunk 2]
```diff
@@ -13,54 +13,57 @@ Couchbase provides several SDKs to allow applications to access a Couchbase clus
...
-
-| Scala SDK
-| Our new Scala SDK allows Scala applications to access a Couchbase Server cluster.
-It offers synchronous, asynchronous, and reactive APIs for flexibility and maximum performance.
-|xref:1.1@scala-sdk:hello-world:overview.adoc[Scala SDK 1.2]
-
 | .NET SDK
 | The .NET SDK enables you to interact with a Couchbase Server cluster from the .NET Framework using any Common Language Runtime (CLR) language, including C#, F#, and VB.NET. 
 It offers both a traditional synchronous API and an asynchronous API based on the Task-based Asynchronous Pattern (TAP).
 []
```
Comment (simon-dew) [inline modules/fts/pages/fts-searching-with-sdk.adoc]: Not sure what the empty square brackets are for. Looks like you could just remove this line.

File: modules/fts/pages/fts-searching-with-sdk.adoc [hunk 3]
```diff
@@ -13,54 +13,57 @@ Couchbase provides several SDKs to allow applications to access a Couchbase clus
...
 |C SDK
 |The Couchbase C SDK (`libcouchbase`) enables C and C++ programs to access a Couchbase Server cluster.
 The C SDK is also commonly used as a core dependency of SDKs written in other languages to provide a common implementation and high performance.
 Libcouchbase also contains the `cbc` suite of command line tools.
-|xref:3.0@c-sdk:hello-world:start-using-sdk.adoc[C SDK 3.2]
+|xref:3.3@c-sdk:howtosfull-text-search.adoc[C SDK 3.3]
+
+|Go SDK
+|The Couchbase Go SDK allows you to connect to a Couchbase Server cluster from Go.
+The Go SDK is a native Go library and uses the high-performance gocbcore to handle communicating to the cluster over Couchbase's binary protocols.
```
Comment (RayOffiah) [inline modules/fts/pages/fts-searching-with-sdk.adoc]:
```suggestion
The Go SDK is a native Go library and uses the high-performance `gocbcore` to handle communicating to the cluster over Couchbase's binary protocols.
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Mostly LGTM, just some minor typos. One question though, I thought the canonical order for SDKs was C first, .NET second &mdash; do I have that wrong?

File: modules/fts/pages/fts-searching-with-sdk.adoc [hunk 4]
```diff
@@ -13,54 +13,57 @@ Couchbase provides several SDKs to allow applications to access a Couchbase clus
...
+
+| Java SDK
+| The Java SDK forms the cornerstone of our JVM clients.
+It allows Java applications to access a Couchbase Server cluster.
+The Java SDK offers traditional synchronous APIs and scalable asynchronous APIs to maximize performance
+|xref:3.3@java-sdk:howtosfull-text-search.adoc[Java SDK 3.3]
+
+| Kotlin SDK
+| Our new Kotlin SDK allows Kotlin applications to access a Couchbase Server cluster.
+It allows Kotlin applications to access a Couchbase Server cluster.
```
Comment (RayOffiah) [inline modules/fts/pages/fts-searching-with-sdk.adoc]: Do you need this line? I think the line above says the same thing, doesn't it?

**Review by RayOffiah — CHANGES_REQUESTED** [review]
I think there's a repeating line in there.

File: modules/fts/pages/fts-searching-with-sdk.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/fts/pages/fts-searching-with-sdk.adoc]: :+1:

File: modules/fts/pages/fts-searching-with-sdk.adoc (hunk 4 above)
Comment (RichardSmedley) [reply modules/fts/pages/fts-searching-with-sdk.adoc]: Fair point.  :+1:

File: modules/fts/pages/fts-searching-with-sdk.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/fts/pages/fts-searching-with-sdk.adoc]: :+1:

**Comment by RichardSmedley** [comment]
Thanks @RayOffiah & @simon-dew  - changes incorporated  :+1: 

The underlying problem with this page is that the general bumf borrowed from the SDK landing page is not useful here - something relating to the SDK's abilities would be much better.

Add to the pile of pages to be improved over the next year?

**Pull request merged**